docs/sources/whatsnew/whats-new-in-v10-0.md

```diff
@@ -0,0 +1,275 @@
+---
+description: Feature and improvement highlights for Grafana v10.0
+keywords:
+  - grafana
+  - new
+  - documentation
+  - '10.0'
+  - release notes
+title: What's new in Grafana v10.0
+weight: -33
+---
+
+# What’s new in Grafana v10.0
+
+Welcome to Grafana 10.0! Read on to learn about changes to search and navigation, dashboards and visualizations, and authentication and security. For even more detail about all the changes in this release, refer to the [changelog](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
+
+<!-- Template below
+## Feature
+[Generally available | Available in experimental/beta] in Grafana [Open Source, Enterprise, Cloud Free, Cloud Pro, Cloud Advanced]
+Description. Include an overview of the feature and problem it solves, and where to learn more (like a link to the docs).
+> **Note:** You must use relative references when linking to docs within the Grafana repo. Please do not use absolute URLs. For more information about relrefs, refer to [Links and references](/docs/writers-toolkit/writing-guide/references/).-->
+
+## Authentication and authorization
+
+### Role-based access control is always enabled
+
```

Comment:
@vtorosyan - Should there be a feature availability note (e.g., _Generally available in all editions of Grafana_)

Reply:
Ahhh okay, I understand. Will review your PR.